deploy/modules/azure/aks/manifests/local-ssd-provision.yaml

```diff
@@ -0,0 +1,200 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: "local-storage"
+provisioner: "kubernetes.io/no-provisioner"
+volumeBindingMode: "WaitForFirstConsumer"
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: local-provisioner-config
+  namespace: kube-system
+data:
+  setPVOwnerRef: "true"
+  nodeLabelsForPV: |
+    - kubernetes.io/hostname
+  storageClassMap: |
+    local-storage:
```

Comment:
`local-storage` is not recommended in production env, because we will have to upgrade ACK from time to time, in which case, we have to migrate the data during each upgrade. 
Please refer to https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions.

Reply:
AKS by default creates storageClasses for example `managed-premium` for us,  the `local-storage` is another option for users.  There are many blockers for adopting remote disks in azure environments, @Ishiihara, Yi Wu and I have been discussing about `local NVMe` vs `remote disks` many rounds.  Both of them have their pros and cons,  I agree that extra copy is needed for local disk migration, however local disk runs much faster than premium disks.  UltraSSD although performs the same as local NVMe but it's not supported in all regions.